Snowflake source freshness (#5228)
dbt-labs/docs.getdbt.com: website/docs/reference/resource-configs/snowflake-configs.md

</VersionBlock>

</VersionBlock>

## Source freshness known limitation

> Snowflake source freshness is calculated using information from the `LAST_ALTERED` column. 

Reviewer: I suggest we combine the last paragraph and the 1st. Maybe something like:
```suggestion
Snowflake calculates source freshness using information from the `LAST_ALTERED` column, meaning it relies on a field updated whenever any object undergoes modification, not only data updates. No action must be taken, but analytics teams should note this caveat. 
```

Per the [Snowflake documentation](https://docs.snowflake.com/en/sql-reference/info-schema/tables#usage-notes): 

  >The `LAST_ALTERED` column is updated when the following operations are performed on an object:
  >- DDL operations.
  >- DML operations (for tables only).
  >- Background maintenance operations on metadata performed by Snowflake.

> This means that Snowflake source freshness is determined by a field that is updated when any object is updated, not just data updates. No action must be taken, but analytics teams should note this caveat. 

Reviewer: ```suggestion
```

Author: Good call!

> This means that Snowflake source frehssness is determined by a field that is updated when any object is updated, not just data updates. There is no action that needs to be taken, but analytics teams should note this caveat. 

Author: ```suggestion
This means that Snowflake source freshness is determined by a field that is updated when any object is updated, not just data updates. No action must be taken, but analytics teams should note this caveat. 
```
